app/services/force/soql/rental_assistance_service.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
```

Comment:
so many new clean and organized services! 🎉

Reply:
😍